views/js/qtiCreator/widgets/interactions/orderInteraction/states/Question.js

```diff
@@ -70,22 +71,6 @@ define([
 
         interactionFormElement.syncMaxChoices(_widget);
 
-        //modify the checkbox/radio input appearances
-        _widget.on('attributeChange', function(data){
```

Comment:
there is no checkbox nor radio input for order interaction, this is inherited from a copy/paste from the choice interaction